src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -170,20 +176,20 @@
 (defn header
   []
   (let [screen-padding 20]
-    ;; TODO: Remove temporary (and old) background color when the screen and
-    ;; header are properly blurred.
-    [rn/view {:background-color (:ui-background @quo.colors/theme)}
-     [button/button {:icon     true
-                     :type     :grey
-                     :size     32
-                     :style    {:margin-vertical 12
-                                :margin-left     screen-padding}
-                     :on-press #(>evt [:navigate-back])}
+    [rn/view
+     [button/button {:icon           true
+                     :type           :blur-bg
+                     :size           32
+                     :override-theme :dark
+                     :style          {:margin-vertical  12
+                                      :margin-left      screen-padding}
+                     :on-press       #(>evt [:hide-popover])}
       :main-icons2/close]
      [text/text {:size   :heading-1
                  :weight :semi-bold
                  :style  {:padding-horizontal screen-padding
-                          :padding-vertical   12}}
+                          :padding-vertical   12
+                          :color              colors/white}}
```

Comment:
does this need a different color depending on theme? 🤔

Reply:
For Activity Center, we force dark mode colours on both themes.